### ISSUE TITLE
Suggest rewording this sentence. 1) the first sentence in section 6.1 has a MAY in it which contradicts the SHALL in this sentence. 

### ISSUE THREAD

**Issue by dig-comments** [usnistgov/800-63-3#802]
**Organization Name (N/A, if individual)**: National Security Agency

**Organization Type**: 1
 
**Document (63-3, 63A, 63B, or 63C)**: 800-63B

**Reference (Include section and paragraph number)**: 6, 1
 
**Comment (Include rationale for comment)**:  Suggest rewording this sentence. 1) the first sentence in section 6.1 has a MAY in it which contradicts the SHALL in this sentence.
 
2) change sentence to read: "This section describes the actions that 'are' taken in response to lifecycle events." this then qualifies what events are covered in the discussion and also removes the SHALL and makes the sentence conform to the style of the other normative sections.

Original text: "This section describes the actions that SHALL be taken in response to those events."

**Suggested Change**: Make suggested change.
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jimfenton** [comment]
The confusing part (see referenced issue) seems to be the use of two MAY clauses to describe possible alternatives. One or the other is required, and that is not well expressed by the current language.
